internal/uvm/scsi.go

```diff
@@ -534,3 +614,12 @@ func (sm *SCSIMount) Clone(ctx context.Context, vm *UtilityVM, cd *cloneData) er
 func (sm *SCSIMount) GetSerialVersionID() uint32 {
 	return scsiCurrentSerialVersionID
 }
+
+// ParseExtensibleVirtualDiskPath parses the evd path provided in the config.
+// extensible virtual disk path has format "evd://<evdType>/<evd-mount-path>"
+// this function parses that and returns the `evdType` and `evd-mount-path`.
+func ParseExtensibleVirtualDiskPath(hostPath string) (evdType, mountPath string) {
+	trimmedPath := strings.TrimPrefix(hostPath, "evd://")
+	separatorIndex := strings.Index(trimmedPath, "/")
```

Comment:
We should probably validate the return from `strings.Index`, otherwise if we get malformed input we will panic when slicing the string.